ec2/InstanceType.dhall

```diff
@@ -543,12 +538,19 @@ let alternatives =
                     let cross-instances = [ InstanceType.r5-large ]
 
                     in    instancefamily
-                        # (       if _params.permitCrossInstanceFamily
-
+                        # ( if    _params.permitCrossInstanceFamily
                             then  cross-instances
-
                             else  [] : List InstanceType
                           )
+                , t3-nano = [ InstanceType.c5-large, InstanceType.c5d-large ]
+                , t3-micro = [ InstanceType.c5-large, InstanceType.c5d-large ]
+                , t3-small = [ InstanceType.c5-large, InstanceType.c5d-large ]
+                , t3-medium = [ InstanceType.c5-large, InstanceType.c5d-large ]
+                , t3-large = [ InstanceType.c5-large, InstanceType.c5d-large ]
+                , t3-xlarge =
+                  [ InstanceType.c5-xlarge, InstanceType.c5d-xlarge ]
+                , t3-2xlarge =
+                  [ InstanceType.c5-2xlarge, InstanceType.c5d-2xlarge ]
```

Comment:
Not the correct set of alternatives.
1) for example a t3.large has 8 GB RAM whereas a c5.large has 4 GB RAM, not as much. Same issue for t3.xlarge and t3.2xlarge
2) you need to check for `_params.permitCrossInstanceFamily` before switching from `t*` to `c*`
3) Ignores `m*`, `r*` options

Reply:
1 - yea, didn't understand the test and what it enforcing. fixed.
2 - I'll add a check
3 - do you mean to add support in these types?

Reply:
No I mean, if a `t3.large` has 2 vCPU and 8 GB RAM then an `m5.large`, which also has 2 vCPU and 8 GB RAM, should be an alternative for the `t3.large`.